### PR TITLE
Quality Audit - Improve light docs

### PR DESCRIPTION
**Pull request by alex-vazquez**
### Purpose of this PR
https://jira.unity3d.com/browse/XPIPELINE-202
Update the light docs with sections

### CODE REVIEW

**Comment by jenniferd-unity** [comment]
hi @oleks-k , do you have time to review this during this week? otherwise, when would you have time? thanks a lot!

File: com.unity.render-pipelines.universal/Documentation~/light-component.md [hunk 1]
```diff
@@ -5,17 +5,58 @@ Lights determine the shading of an object and the shadows it casts.
 This page contains information on Light components in the Universal Render Pipeline (URP). For a general introduction to lighting in Unity and examples of common lighting workflows, see [the Lighting section of the Unity Manual](https://docs.unity3d.com/Manual/LightingOverview.html).
 
 ## Properties
+The Light Inspector includes the following groups of properties:
+
 ![](Images/Inspectors/light-inspector.png)
 
+* [General](#General)
+* [Emission](#Emission)
```
Comment (reallyfancy) [inline com.unity.render-pipelines.universal/Documentation~/light-component.md:13]: Emission and shape are either in the wrong order in this list, or in the wrong order on the page. Either way, these links should match the order the information appears in!

**Review by reallyfancy — APPROVED** [review]
One minor suggested change.

**Review by erikabar — APPROVED** [review]
looks good

**Pull request merged**